hr_cae/models/hr_employee.py

```diff
@@ -31,14 +38,21 @@ class Employee(models.Model):
     )
     certificate_id = fields.Many2one(
         "hr.recruitment.degree", string="Certificate", required=False
-    )  # TODO: copy from applicant_id.type_id on creation
+    )
+    certificate_date = fields.Date(
+        string="Certificate Date", help="Certificate Delivery Date", required=False
+    )
+    professional_experience = fields.Text(
+        string="Professional Experience", required=False
+    )
     bank_account_payment_id = fields.Many2one(
         "res.partner.bank",
         string="Bank Account Number for Payment",
         required=False,
         domain="[('partner_id', '=', address_home_id)]",
         help="Employee bank salary account",
     )  # TODO: check that this one is actually used in logic
+    turnover_minimum = fields.Monetary(string="Minimum Turn-Over")
```

Comment:
Agreed, I'm not sure to understand the logic behind this field. Not blocking for me

Reply:
This is a field where CAE members can indicate the turnover they will try to make.